app/helpers/records_helper.rb

```diff
@@ -43,7 +34,11 @@ def filter_class(filter_name)
   end
 
   def date_parser_for(string)
-    Date.parse string rescue return string
+    begin
+      Date.parse string
```

Comment:
Can just move `:42` to in place of `:38`

Wait just realised these are cop autocorrects - so don't worry if you just want to leave the logic as is

Reply:
Yeah I didn't change this myself... I am happy if rubocop is happy most of the time ^^